profiler/test/Datadog.Profiler.IntegrationTests/Helpers/TestApplicationRunner.cs

```diff
@@ -98,7 +118,7 @@ private string GetApplicationPath()
             return Path.Combine(GetApplicationOutputFolderPath(_appName), _framework, GetApplicationAssemblyFileName());
         }
 
-        private (string Executor, string Arguments) BuildTestCommandLine()
+        private (string Executor, string Arguments) BuildTestCommandLine(bool catchsegv = true)
```

Comment:
This has changed, we do not rely on `catchsegv` anymore

Reply:
Fixed, thanks